content/docs/run-api-server/installing.mdx

```diff
@@ -5,25 +5,115 @@ order: 20
 
-After building or unpacking Horizon, you simply need to copy the native binary into a directory that is part of your PATH. Most unix-like systems have `/usr/local/bin` in PATH by default, so unless you have a preference or know better, we recommend you copy the binary there.
+## Installation Methods
 
-To test the installation, simply run `horizon --help` from a terminal. If the help for Horizon is displayed, your installation was successful. Note: some shells, such as zsh, cache PATH lookups. You may need to clear your cache (by using `rehash` in zsh, for example) before trying to run `horizon --help`.
+To install Horizon, you have a few choices. You can...
+  - install prebuilt binaries [from our repositories](#package-manager) via your package manager if running a Debian-based system,
+  - download a [prebuilt release](https://github.com/stellar/go/releases) for your target architecture and operation system, or
+  - [build Horizon yourself](#building) from scratch. 
 
-## Building
+The first method is recommended: not only do you ensure OS compatibility and dependency management, you'll also install some convenient wrappers that make running Horizon and Stellar-Core much simpler.
 
+### Package Manager
+SDF publishes new releases to its custom Ubuntu repositories. Follow [this guide](https://github.com/stellar/packages/blob/master/docs/adding-the-sdf-stable-repository-to-your-system.md#adding-the-sdf-stable-repository-to-your-system) to add the stable SDF repository to your system. Then, 
+
+<CodeExample>
+
+```bash
+sudo apt update && sudo apt install stellar-captive-core stellar-horizon
+```
+
+</CodeExample>
+
+[This page](https://github.com/stellar/packages/blob/master/docs/installing-individual-packages.md#installing-individual-packages) outlines the various commands that these packages make available.
+
+### Building
 Should you decide not to use one of our prebuilt releases, you may instead build Horizon from source. To do so, you need to install some developer tools:
 
-- A unix-like operating system with the common core commands (cp, tar, mkdir, bash, etc.)
+- A Unix-like operating system with the common core commands (cp, tar, mkdir, bash, etc.)
 - A compatible distribution of Go (Go 1.13 or later)
 - [git](https://git-scm.com/)
-- [mercurial](https://www.mercurial-scm.org/)
 
-1. See the details in [README.md](https://github.com/stellar/go/blob/master/README.md#dependencies) for installing dependencies.
-2. Compile the Horizon binary: `go install github.com/stellar/go/services/horizon`. You should see the `horizon` binary in `$GOPATH/bin`.
-3. Add Go binaries to your PATH in your `bashrc` or equivalent, for easy access: `export PATH=${GOPATH//://bin:}/bin:$PATH`
+**Note**: Though Horizon should run on Windows, *building* directly on Windows is not supported.
```

Comment:
```suggestion
**Note**: Though Horizon can run on Windows, *building* directly on Windows is not supported.
```

Reply:
To avoid confusion since this can be read as "You should run Horizon on Windows."